internal/common/config/helpertypes.go

```diff
@@ -35,3 +35,20 @@ func getSelectivelyVisibleResources(value shared.Resources, valid bool) Selectiv
 		Valid: valid,
 	}
 }
+
+// SelectivelyVisibleInteger represents a wrapped int whose value is valid only if the Valid flag is set to true
```

Comment:
i prefer `SelectivelyVisibleInt` but up to you :)

Reply:
Yeah that sounds better tbh